src/librustc_mir/dataflow/impls/mod.rs

```diff
@@ -243,34 +243,37 @@ impl<'a, 'gcx, 'tcx> HasMoveData<'tcx> for EverInitializedPlaces<'a, 'gcx, 'tcx>
 
 
 impl<'a, 'gcx, 'tcx> MaybeInitializedPlaces<'a, 'gcx, 'tcx> {
-    fn update_bits(sets: &mut BlockSets<'_, MovePathIndex>, path: MovePathIndex,
+    fn update_bits(trans: &mut GenKillSet<MovePathIndex>,
+                   path: MovePathIndex,
                    state: DropFlagState)
     {
         match state {
-            DropFlagState::Absent => sets.kill(path),
-            DropFlagState::Present => sets.gen(path),
+            DropFlagState::Absent => trans.kill(path),
+            DropFlagState::Present => trans.gen(path),
         }
     }
 }
 
 impl<'a, 'gcx, 'tcx> MaybeUninitializedPlaces<'a, 'gcx, 'tcx> {
```

Comment:
```suggestion
impl MaybeUninitializedPlaces<'_, '_, '_> {
```

Reply:
I'll probably leave these alone since they will get changed anyway when the `TyCtxt` -> `TyCx` renaming occurs and having actual names will make it easier to search/replace.

Reply:
Yeah, fair.